frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/storage-tab-initial-state.ts

```diff
@@ -111,6 +168,29 @@ export const getNewProvisionSourceStorage = (state: any, id: string): VMWizardSt
   if (provisionSource === ProvisionSource.CONTAINER) {
     return containerStorage;
   }
+  if (provisionSource === ProvisionSource.DISK && cloneCommonBaseDiskImage) {
+    const iStorageClassConfigMap = iGetLoadedCommonData(
+      state,
+      id,
+      VMWizardProps.storageClassConfigMap,
+    );
+
+    const relevantOptions = iGetRelevantTemplateSelectors(state, id);
+    if (!relevantOptions.os) {
+      return null;
+    }
+
+    const iCommonTemplates = iGetLoadedCommonData(state, id, VMWizardProps.commonTemplates);
+    const iTemplate =
+      iCommonTemplates && iGetRelevantTemplate(null, iCommonTemplates, relevantOptions);
```

Comment:
the same here, not acknowledging use templates could lead to incorrect result (I suppose it should be enough to make cloneCommonBaseDiskImage consistent). But please write a comment about it here

Reply:
done

Reply:
please write a comment that cloneCommonBaseDiskImage is set true only if userTemplate is not used

IMO better documentation would be actually to check the userTemplate name and return early